src/bidiMapper/BidiServer.ts

```diff
@@ -85,9 +85,7 @@ export class BidiServer extends EventEmitter<BidiServerEvents> {
       flatten: true,
     });
 
-    await Promise.all(
-      BrowsingContextStorage.getTopLevelContexts().map((c) => c.awaitLoaded())
-    );
+    await Promise.all(getTopLevelContexts().map((c) => c.awaitLoaded()));
```

Comment:
this data should be local to the server. Let's not migrate the static until we address the enderlying issue? WDYT? or let's mark it somehow with a TODO so that we can find it (currently, the static keyword is an indicator of the issue.).

Reply:
Ack. Will rework this PR